### DISCUSSION TITLE
plz help us

### DISCUSSION THREAD

**Discussion by AsaphIF** [mobizt/ESP-Google-Sheet-Client#3] [Q&A] [answered]
Hi Mobizt, how are you?
I am doing a project with my bro, in ESP32
I am using your library <ESP_Google_Sheet_Client.h>
and everything is going well with the program 
the program can read the text in google sheets
But I need help
I can't transform 'response.toString(Serial,true);' in a String
I need to get some information from text in response.toString(Serial,true);
but I can't 
I know response is FirebaseJson but Idk how to fix it
'FirebaseJson response;'
I tried

const char *str = response.toString(Serial,true).c_str();
but no results

I am using an ESP32 DEVKIT V1
Arduino IDE

if you can help me I'll be Thankfull

**Comment by mobizt** [comment, marked as answer]
The FirebaseJson functions are incorrectly used in your code.

You should learn how to use it from [this example](https://github.com/mobizt/ESP-Google-Sheet-Client/blob/master/examples/FirebaseJson/BasicUsage/Create_Edit_Parse/Create_Edit_Parse.ino).

**Reply by mobizt** [reply]
I don't have time to debug your code and you should do it yourself.

**Reply by AsaphIF** [reply]
thanks
